__tests__/__integration__/cli/__snapshots__/cli.mqsc.integration.test.ts.snap

```diff
@@ -54,6 +54,8 @@ exports[`mq mqsc should display the mqsc help 1`] = `
 
       Reject self-signed certificates.
 
+      Default value: false
```

Comment:
Is it OK for this setting to be insecure by default? (I know this wasn't a change introduced in this PR, just wondering.)

Reply:
Probably not? This may be considered an acceptable (slightly-breaking) change to do though.
@MikeBauerCA, @Colin-Stone, thoughts?